src/ConsoleMode.ps1

```diff
@@ -0,0 +1,103 @@
+# Hack! https://gist.github.com/lzybkr/f2059cb2ee8d0c13c65ab933b75e998c
+
+if ($IsWindows -eq $false) {
```

Comment:
I guess I find the logic a bit cryptic.   Maybe this is worth a helper function IsWindows to encapsulate the logic.  RE `NativeConsoleMethods` no need to create these if they are not used and we won't use them except on Windows w/ANSI support, right?